--- Makefile.vc
+++ Makefile.vc
@@ -76,7 +76,6 @@
   $(LIBGD_OBJ_DIR)\gd_jpeg.obj \
   $(LIBGD_OBJ_DIR)\gd_security.obj \
   $(LIBGD_OBJ_DIR)\gd_tga.obj \
-  $(LIBGD_OBJ_DIR)\gd_tiff.obj \
   $(LIBGD_OBJ_DIR)\gd_xbm.obj \
   $(LIBGD_OBJ_DIR)\gdkanji.obj \
   $(LIBGD_OBJ_DIR)\gd_nnquant.obj \
@@ -93,13 +92,10 @@
   $(LIBGD_OBJ_DIR)\gd_rotate.obj \
   $(LIBGD_OBJ_DIR)\gd_version.obj \
   $(LIBGD_OBJ_DIR)\gd_crop.obj \
-  $(LIBGD_OBJ_DIR)\gd_color_map.obj \
-  $(LIBGD_OBJ_DIR)\gd_heif.obj \
-  $(LIBGD_OBJ_DIR)\gd_avif.obj \
-  $(LIBGD_OBJ_DIR)\gd_webp.obj
+  $(LIBGD_OBJ_DIR)\gd_color_map.obj
 
 LIBS=kernel32.lib ole32.lib user32.lib advapi32.lib shell32.lib ws2_32.lib Dnsapi.lib Gdi32.Lib
-LIBS_GD=libjpeg_a.lib freetype_a.lib libpng_a.lib libiconv_a.lib  zlib_a.lib libheif_a.lib libwebp_a.lib libxpm_a.lib libavif_a.lib libtiff.lib
+LIBS_GD=libjpeg_a.lib freetype_a.lib libpng_a.lib zlib_a.lib
 
 PROG_EXES= \
 	$(LIBGD_OBJ_DIR)\gdcmpgif.exe \
@@ -116,15 +112,15 @@
 
 INCLUDES=/Isrc /I$(WITH_DEVEL)\include -I$(WITH_DEVEL)\include\libpng16 -I$(WITH_DEVEL)\include\freetype -I$(WITH_DEVEL)\include\freetype2 /Itests /Itests\gdtest -I$(WITH_DEVEL)\include\
 
-LDFLAGS=/nologo /nodefaultlib:libcmt /libpath:$(WITH_DEVEL)\lib /MACHINE:$(PLATFORM)
+LDFLAGS=/nologo /libpath:$(WITH_DEVEL)\lib /MACHINE:$(PLATFORM)
 !IF $(DEBUG)
 LDFLAGS=$(LDFLAGS) /DEBUG
 !ENDIF
 
 !IF $(DEBUG)
-CFLAGS=/nologo /Qvec-report:1 /MD /TC /Zi
+CFLAGS=/nologo /Qvec-report:1 /MTd /TC /Zi
 !ELSE
-CFLAGS=/nologo /O2 /Qvec-report:1 /MD /TC
+CFLAGS=/nologo /O2 /Qvec-report:1 /MT /TC
 !ENDIF
 
 CFLAGS= $(CFLAGS) \
@@ -135,17 +131,10 @@
        /DENABLE_GD_FORMATS=1\
        /DHAVE_FT2BUILD_H=1\
        /DHAVE_GD_H=1\
-       /DHAVE_ICONV_H=1\
-       /DHAVE_LIBAVIF=1\
        /DHAVE_LIBFREETYPE=1\
        /DHAVE_LIBJPEG=1\
        /DHAVE_LIBPNG=1\
-       /DHAVE_LIBWEBP=1\
-       /DHAVE_LIBHEIF=1\
-       /DHAVE_LIBAVIF=1\
        /DHAVE_LIBZ=1\
-       /DHAVE_LIBXPM=1\
-       /DHAVE_LIBTIFF=1\
        /DHAVE_CONFIG_H\
        $(INCLUDES)
 
@@ -187,12 +176,6 @@
 	@echo #ifndef HAVE_FT2BUILD_H> $(GD_CONFIG_H)
 	@echo #define HAVE_FT2BUILD_H>> $(GD_CONFIG_H)
 	@echo #endif>> $(GD_CONFIG_H)
-	@echo #ifndef HAVE_ICONV_H>> $(GD_CONFIG_H)
-	@echo #define HAVE_ICONV_H>> $(GD_CONFIG_H)
-	@echo #endif>> $(GD_CONFIG_H)
-	@echo #ifndef HAVE_LIBAVIF>> $(GD_CONFIG_H)
-	@echo #define HAVE_LIBAVIF>> $(GD_CONFIG_H)
-	@echo #endif>> $(GD_CONFIG_H)
 	@echo #ifndef HAVE_LIBFREETYPE>> $(GD_CONFIG_H)
 	@echo #define HAVE_LIBFREETYPE>> $(GD_CONFIG_H)
 	@echo #endif>> $(GD_CONFIG_H)
@@ -202,30 +185,15 @@
 	@echo #ifndef HAVE_LIBPNG>> $(GD_CONFIG_H)
 	@echo #define HAVE_LIBPNG>> $(GD_CONFIG_H)
 	@echo #endif>> $(GD_CONFIG_H)
-	@echo #ifndef HAVE_LIBWEBP>> $(GD_CONFIG_H)
-	@echo #define HAVE_LIBWEBP>> $(GD_CONFIG_H)
-	@echo #endif>> $(GD_CONFIG_H)
-	@echo #ifndef HAVE_LIBHEIF>> $(GD_CONFIG_H)
-	@echo #define HAVE_LIBHEIF>> $(GD_CONFIG_H)
-	@echo #endif>> $(GD_CONFIG_H)
-	@echo #ifndef HAVE_LIBAVIF>> $(GD_CONFIG_H)
-	@echo #define HAVE_LIBAVIF>> $(GD_CONFIG_H)
-	@echo #endif>> $(GD_CONFIG_H)
 	@echo #ifndef HAVE_LIBZ>> $(GD_CONFIG_H)
 	@echo #define HAVE_LIBZ>> $(GD_CONFIG_H)
 	@echo #endif>> $(GD_CONFIG_H)
-	@echo #ifndef HAVE_LIBXPM>> $(GD_CONFIG_H)
-	@echo #define HAVE_LIBXPM>> $(GD_CONFIG_H)
-	@echo #endif>> $(GD_CONFIG_H)
 	@echo #ifndef HAVE_INTTYPES_H>> $(GD_CONFIG_H)
 	@echo #define HAVE_INTTYPES_H>> $(GD_CONFIG_H)
 	@echo #endif>> $(GD_CONFIG_H)
 	@echo #ifndef HAVE_SYS_STAT_H>> $(GD_CONFIG_H)
 	@echo #define HAVE_SYS_STAT_H>> $(GD_CONFIG_H)
 	@echo #endif>> $(GD_CONFIG_H)
-	@echo #ifndef HAVE_LIBTIFF>> $(GD_CONFIG_H)
-	@echo #define HAVE_LIBTIFF>> $(GD_CONFIG_H)
-	@echo #endif>> $(GD_CONFIG_H)
 	@echo #define ssize_t SSIZE_T>> $(GD_CONFIG_H)
 	@echo #define SSIZE_MAX MAXSSIZE_T>> $(GD_CONFIG_H)
 
